javascript/react/src/components/verbs/descriptions/CopyDescription.tsx

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { memo, useMemo } from 'react'
+
+import type { StepDescriptionProps } from './types.js'
+import { VerbDescription } from './VerbDescription.js'
+
+export const CopyDescription: React.FC<StepDescriptionProps<void>> = memo(
```

Comment:
I tried it but the difference is the no parameters form verbs don't have an output column ("to" parameter) so it doesn't show the "to" field for the user. So that's the reason why I couldn't use the no parameters form.

Reply:
I think the output column should be automatically added to the form if you have tagged the verb with Tags.OutputColumn